app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -204,7 +112,7 @@ private function parseReferralCode($referralCode)
                     'northstar_id' => null, // set the user to null so we force account creation when the code is not present.
                     'campaign_id' => $values['campaign_id'],
```

Comment:
I am 🤔why a post is created for 8017 instead of 8167 in this test file Tracking Code`user:null,campaign:8167,campaignRunId:8168,source:web,source_details:PollCallActionPage `.

I'm fairly certain #61 is throwing an error for me locally because it's trying to create a post for campaign 8167. I'm also wondering if some of this logic has been deprecated and never removed?

Reply:
@aaronschachter hm this is weird! Are you testing on QA? We don't do anything with `campaign_run_id` in Rogue anymore, just look for `campaign_id`...